wiki/Performance_points/id.md

```diff
@@ -0,0 +1,132 @@
+
+### Pembidikan
+
+*Pembidikan* adalah nilai inti yang mempertimbangkan betapa sulitnya mengenai not secara berurutan dan konsisten dalam sebuah beatmap.
+
+Elemen seperti [approach rate](/wiki/Beatmapping/Approach_rate) dan [mods](/wiki/Game_modifier) tertentu (yaitu [Flashlight](/wiki/Game_modifier/Flashlight), [Hidden](/wiki/Game_modifier/Hidden) dan [Hard Rock](/wiki/Game_modifier/Hard_Rock)) membuat menavigasi kursor dengan cepat dan akurat jauh lebih sulit, dan dengan demikian memengaruhi jumlah pp yang diberikan oleh skor tersebut.
+
+Dalam kasus seperti [osu!standard](/wiki/Game_mode/osu!), beatmap dengan [jumps](/wiki/Beatmap/Pattern/Jump) yang sangat besar itu dipertimbangkan sebagai beatmaps "bidikan tinggi", dan oleh karena itu, sering memberikan skor pp yang sangat tinggi. Demikian juga, beatmaps dengan lebih banyak hyperdashing di [osu!catch](/wiki/Game_mode/osu!catch) itu akan dipertimbangkan dengan sama. Pidikan itu tidak dipertimbangkan didalam gamemode seperti [osu!taiko](/wiki/Game_mode/osu!taiko) dan [osu!mania](/wiki/Game_mode/osu!mania).
+
+### Kecepatan
+
+*Kecepatan* adalah nilai inti yang mempertimbangkan pada nilai mana sebuah menyajikan elemen untuk play tersebut.
+
+Beatmap dengan jumlah hit objek yang tinggi dalam waktu yang singkat juga dipertimbangkan untuk mempunyai jumlah kecepatan yang tinggi. Dalam aspek spesifik ini, semakin cepat kecepatan beatmap itu, semakin sulit beatmap tersebut itu menjadi, karena itu, memberikan jumlah pp yang diperoleh lebih besar.
+
+Sebagai hasil dari ini, mod seperti [Double Time](/wiki/Game_modifier/Double_Time) dan [Half Time](/wiki/Game_modifier/Half_Time) secara signifikan mempengaruhi kecepatan beatmap yang dipertimbangkan oleh algoritma poin performa. Demikian juga, mod-mod itu juga secara signifikan mempengaruhi jumlah pp yang diperoleh saat digunakan.
+
+### Akurasi
+
+*Lihat juga: [Akurasi](/wiki/Accuracy)*
+
+*Akurasi* adalah sebuah pengukuran persen dari kemampuan pemain untuk mengenai [hit objek](/wiki/Hit_object) pada waktunya; Berhubungan dengan algoritma pp, itu juga adalah sebuah nilai inti yang digunakan untuk mengevaluasi kinerja individu pemain pada sebuah beatmap.
+
+Skor dengan jumlah akurasi yang tinggi itu dipertimbangkan oleh algoritma tersebut sebagai yang paling mengagumkan, dan juga akan memberikan skor yang sangat besar oleh karena ini. Sebuah play [full combo](/wiki/Glossary#fc) yang diraih dengan 80% akurasi kadang-kadang bisa hanya senilai 2/3 dari skor yang diraih dengan 95% akurasi. Karena algoritma ini sangat bergantung pada akurasi, mod seperti Hidden, Hard Rock dan Flashlight itu dipertimbangkan untuk meningkatkan jumlah pp tersebut dengan akuraai yang tinggi secara signifikan.
+
+### Ketegangan
+
+*Ketegangan* adalah nilai inti yang mempertimbangkan berapa kali, dan berapa lama, seorang pemain bisa mengikuti bagian yang intensitas-nya tinggi dalam beatmap tertentu.
```

Comment:
related to http://github.com/ppy/osu-wiki/pull/5119#issuecomment-806474761 :

`` ### Strain

*Strain* merupakan variabel perhitungan pp yang menyangkut seberapa mahir seorang pemain dalam memainkan bagian-bagian beatmap yang berintensitas tinggi...``

Reply:
done